runtime/vm/description.c

```diff
@@ -164,18 +164,25 @@ calculateInstanceDescription( J9VMThread *vmThread, J9Class *ramClass, J9Class *
 	 * set the corresponding description bit for each 
 	 */
 	{
+		BOOLEAN inheritedSelfReferencingFields = ramClass->selfReferencingField1 != 0 ? TRUE : FALSE;
```

Comment:
Please add a comment to this indicating that it's checking for selfReferencingFields from the superclass.  

Or Is there a way to write this that is more explicit?  Maybe something like this that explictly checks the superclass ramClass->superClasses[depth-1]->selfReferencingField1 != 0?

Reply:
Made this explicit... Checking against `ramSuperClass` instead now. To do so, I needed to move this check a little above, under `if (ramSuperClass)` block and declare the `BOOLEAN` at the beginning of the function.

Reply:
Thanks @RSalman.  This is much clearer